ember-app/app/controllers/issue/create.js

```diff
@@ -5,14 +5,19 @@ var IssuesCreateController = Ember.ObjectController.extend({
   actions: {
     submit: function() {
       this.createIssue(this.get("order"));
+    },
+    assignRepo: function(repo){
+      this.set("model.repo.full_name", repo.full_name);
+      this.set("otherLabels", repo.other_labels);
+      this.set("model.labels", []);
+      this.set("milestones", repo.milestones);
+      this.set("assignees", repo.assignees);
```

Comment:
Hmmm should we go the extra mile and "try" to match up the equivelent selections?